amxmodx/natives_handles.h

```diff
@@ -111,7 +104,6 @@ class NativeHandle
 				return false;
 			}
 
-			delete m_handles[handle];
```

Comment:
Unless I'm wrong, this is still needed here, since the slots are reused, you are going to leak memory.